lib/impress.sse.js

```diff
@@ -136,7 +136,7 @@ impress.sse.mixin = (application) => {
     const connections = application.sse.connections;
     let connectionId, res;
     for (connectionId in connections) {
```

Comment:
We shouldn't had used a for-in loop before these changes because of performance issues, but now this won't even work correctly.